cockroachcloud/cloud-api.md

```diff
@@ -324,9 +324,9 @@ Where:
 - `{region_name}` is the cloud infrastructure provider region where the cluster is located.
 - `{status}` is the status of the node. Possible values are: `LIVE` and `NOT_READY`.
 
-## Set the maximum spend limit of a Serverless cluster
+## Set the maximum resource limit of a Serverless cluster
 
-To set the maximum spend limit for a Serverless cluster, send a `PUT` request to the `/v1/clusters/{cluster_id}/spend-limit` endpoint. The service account associated with the secret key must have `ADMIN` or `EDIT` [permission](console-access-management.html#service-accounts) to retrieve information about an organization's clusters.
+To set the maximum resource limit for a Serverless cluster, send a `PUT` request to the `/v1/clusters/{cluster_id}/spend-limit` endpoint. The service account associated with the secret key must have `ADMIN` or `EDIT` [permission](console-access-management.html#service-accounts) to retrieve information about an organization's clusters.
```

Comment:
I think most of the instructional updates for setting limits should be a follow-up PR with the release. Excepting the main "Create a Serverless cluster" and quickstart docs because they're higher traffic/might be browsed by someone just interested in serverless.